### ISSUE TITLE
Support bounties via Issuehunt

### ISSUE THREAD

**Issue by spoorun** [humhub/humhub#4948]
It would be great if Humhub repositories signed up for Issuehunt support

https://issuehunt.io/r/new

That way people can post bounties in individual bugs they care about and hopefully accelerate development in areas that might be outside the core team short term goals, it will also increase motivation for those involved in Humhub opensource and improve the product.

**Comment by ArchBlood** [comment]
Not related to issuehunt.io but there is a Bounty Source that is up, it's not really used much but it's there

https://www.bountysource.com/teams/humhub

**Comment by spoorun** [comment]
> Not related to issuehunt.io but there is a Bounty Source that is up, it's not really used much but it's there
> 
> https://www.bountysource.com/teams/humhub

Yeah that's cool. Though IssueHunt seems to be easier to track in Github and better integrated

**Comment by samuk** [comment]
Or gitcoin looks powerful https://gitcoin.co/how/funder
